freqtrade/exchange/exchange.py

```diff
@@ -8,9 +8,10 @@
 import signal
 from copy import deepcopy
 from datetime import datetime, timedelta, timezone
+import arrow
```

Comment:
We've intentionally moved away from using arrow anywhere except in reporting - and i don't see us readding this here unless there's a very compelling reason.

it's a lot slower than the builtin datetime methods - and i can see no scenario where arrow would need to be used within the exchange class, where either datetime directly, or the datetime_helpers (see utils directory) wouldn't suffice.